build_scripts/build_runner.py

```diff
@@ -771,46 +770,49 @@ def _strip_bins(self):
         system_os = platform.system()
 
         if system_os == 'Linux':
-            search_bin_results = self.options['ROOT_DIR'].glob('**/__cmake/**/__bin/**/*')
-            search_lib_results = self.options['ROOT_DIR'].glob('**/__cmake/**/__lib/**/*')
-
-            search_results = itertools.chain(search_bin_results, search_lib_results)
-
+            bins_to_strip = []
             binaries_with_error = []
-
-            for result in search_results:
-                if result.is_file():
-                    orig_file = str(result.absolute())
-                    debug_file = str((result.parent / f'{result.stem}.sym').absolute())
-
-                    self.log.info('-' * 80)
-                    self.log.info(f'Stripping {orig_file}')
-
-                    strip_commands = OrderedDict([
-                        ('copy_debug', ['objcopy',
-                                        '--only-keep-debug',
-                                        orig_file,
-                                        debug_file]),
-                        ('strip', ['strip',
-                                   '--strip-debug',
-                                   '--strip-unneeded',
-                                   '--remove-section=.comment',
-                                   orig_file]),
-                        ('add_debug_link', ['objcopy',
-                                            f'--add-gnu-debuglink={debug_file}',
-                                            orig_file]),
-                        ('set_chmod', ['chmod',
-                                       '-x',
-                                       debug_file])
-                    ])
-
-                    for command in strip_commands.values():
-                        err, out = cmd_exec(command, shell=False, log=self.log)
-                        if err:
-                            if orig_file not in binaries_with_error:
-                                binaries_with_error.append(orig_file)
-                            self.log.error(out)
-                            continue
+            excluded_extensions = ['.bin', '.out', '.pc']
```

Comment:
Probably should include executable binaries with no extension only
and binaries with .so, .a extensions
?